translations/monero-core_eo.ts

```diff
@@ -177,33 +185,38 @@
     <message>
         <location filename="../pages/History.qml" line="135"/>
         <source>Filter transaction history</source>
-        <translation>Filtru transakcia historio</translation>
+        <translation>Filtri la transakcian historion</translation>
     </message>
     <message>
         <location filename="../pages/History.qml" line="70"/>
         <source> selected: </source>
         <translation> selektita: </translation>
     </message>
     <message>
+        <location filename="../pages/History.qml" line="147"/>
```

Comment:
Doesn't match [History.qml#L147](https://github.com/monero-project/monero-core/blob/master/pages/History.qml#L147) -- same as the new message added for AddressBook.qml above. (This sentence as well was only found in Hebrew)

Reply:
Same thing, sorry I'm not sure I can help you here.

Reply:
I think I copied the Dutch translation to create the Esperanto translation. No clue what this line is. Should it be deleted?

Reply:
The entire message (everything from lines 195 - 199 inclusive) can be deleted, but not a single line as this will break the XML.